db/database.go

```diff
@@ -258,7 +258,10 @@ func NewDatabaseContext(dbName string, bucket base.Bucket, autoImport bool, opti
 
 	}
 
-	go context.watchDocChanges()
+	if !context.UseXattrs() {
+		go context.watchDocChanges()
+	}
+
```

Comment:
watchDocChanges is only called from here, and it seemed appropriate to avoid the overhead of spawning the goroutine if it wasn't going to be needed.